pkg/features/features.go

```diff
@@ -74,6 +74,12 @@ const (
 	// owner: @hown3d
 	// alpha: v1.130.0
 	UseUnifiedHTTPProxyPort featuregate.Feature = "UseUnifiedHTTPProxyPort"
+
+	// VPAInPlaceUpdates enables the usage of in-place Pod resource updates in the Vertical Pod Autoscaler deployments.
+	// resources to perform in-place Pod resource updates.
```

Comment:
The sentences sound odd and not related to each other.

Did you mean?
```suggestion
	// VPAInPlaceUpdates enables the usage of in-place Pod resource updates in the Vertical Pod Autoscaler resources
	// to perform in-place Pod resource updates.
```

Reply:
This comment is not yet addressed.

Reply:
Committed the suggestion, thanks.